src/generic/UnivPoly.jl

```diff
@@ -900,7 +832,10 @@ function (a::Union{MPolyRingElem, UniversalPolyRingElem})(;kwargs...)
    vals = Array{RingElement}(undef, length(kwargs))
```

Comment:
This is not correct anymore as the number of kwargs might be greater. I'll look into it.